src/Features/Core/Portable/GenerateMember/GenerateParameterizedMember/AbstractGenerateParameterizedMemberService.CodeAction.cs

```diff
@@ -86,12 +86,14 @@ protected override async Task<Document> GetChangedDocumentAsync(CancellationToke
                         _document.Project.Solution,
                         _state.TypeToGenerateIn,
                         method,
-                        new CodeGenerationOptions(afterThisLocation: _state.Location, parseOptions: syntaxTree.Options),
+                        new CodeGenerationOptions(afterThisLocation: _state.Location, generateMethodBodies: !IsInterface(_state.TypeToGenerateIn), parseOptions: syntaxTree.Options),
                         cancellationToken)
                         .ConfigureAwait(false);
 
                     return result;
                 }
+
+                bool IsInterface(INamedTypeSymbol symbol) => symbol.TypeKind == TypeKind.Interface;
             }
```

Comment:
I would just inline this.  Or make an isymbol extension and update everyone to use that